### PR TITLE
Toolranks support and localization

### PR DESCRIPTION
**Pull request by adikalon**

### CODE REVIEW

File: craftitems.lua [hunk 1]
```diff
@@ -1,89 +1,89 @@
 minetest.register_craftitem("magic_materials:arcanite_crystal", {
-    description = "Arcanite Crystal",
+    description = magic_materials.S("Arcanite Crystal"),
```
Comment (Niklp09) [inline craftitems.lua]: Please use `S()` when calling the translation API to reduce repetition.
```lua
local S = magic_materials.S
-- ......
description = S("str")
```

**Review by Niklp09 — COMMENTED** [review]
Please generate a `template.txt` file with https://github.com/minetest/modtools/blob/main/mod_translation_updater.py, it can format the `.ru.tr` file as well.

Everything else LGTM.

File: craftitems.lua (hunk 1 above)
Comment (adikalon) [reply craftitems.lua]: Corrected

Comment (adikalon) [reply craftitems.lua]: Translations also

**Review by Niklp09 — APPROVED** [review]
LGTM. Thank you for contributing!

**Pull request merged**